src/storage/mvcc/mod.rs

```diff
@@ -59,9 +59,16 @@ impl TsSet {
         }
     }
 
-    #[cfg(test)]
     pub fn from_u64s(ts: Vec<u64>) -> Self {
-        Self::new(ts.into_iter().map(Into::into).collect())
+        // This conversion is safe because TimeStamp is a transparent wrapper over u64.
+        let ts = unsafe { ::std::mem::transmute::<Vec<u64>, Vec<TimeStamp>>(ts) };
+        Self::new(ts)
+    }
```

Comment:
It seems that this requires that the `TimeStamp` is the same with u64 im memory, which relies on `TimeStamp`'s internal implementation. How do you think?

Reply:
I think it is OK - it would be hard to change TimeStamp without causing a compile error and TsSet seems very closely related to TimeStamp. I could move the Vec<u64> -> Vec<TimeStamp> conversion to TimeStamp if you prefer that?